src/Organic/AdConfigSyncCommand.php

```diff
@@ -49,8 +49,11 @@ public function run() {
      * @return void
      */
     public function __invoke( $args ) {
-        // Only both trying if the API key is set
-        if ( ! $this->organic->isEnabledAndConfigured() ) {
+        // Only worth trying if Ads is enabled and the API key is set
```

Comment:
I think Ads needs to be enabled, but maybe not?

Reply:
(E.g., maybe similar to Ads.txt redirect not needing Ads enabled)

Reply:
Yeah, I think it worth syncing to make sure we have config before enabling Ads